examples/src/main/java/io/numaproj/numaflow/examples/mapstream/flatmapstream/FlatMapStreamFunction.java

```diff
@@ -1,10 +1,6 @@
 package io.numaproj.numaflow.examples.mapstream.flatmapstream;
 
-import io.numaproj.numaflow.mapstreamer.Datum;
-import io.numaproj.numaflow.mapstreamer.MapStreamer;
-import io.numaproj.numaflow.mapstreamer.Message;
-import io.numaproj.numaflow.mapstreamer.OutputObserver;
-import io.numaproj.numaflow.mapstreamer.Server;
+import io.numaproj.numaflow.mapstreamer.*;
```

Comment:
nit: consider not using wildcard. https://tharakamd-12.medium.com/is-it-bad-to-use-wildcard-imports-in-java-1b46a863b2be